packages/flutter/lib/src/material/slider_theme.dart

```diff
@@ -293,6 +293,9 @@ class SliderThemeData with Diagnosticable {
     this.thumbSelector,
     this.mouseCursor,
     this.allowedInteraction,
+    this.barThumbSize,
+    this.trackGapSize,
+    this.use2024SliderShapes,
```

Comment:
What about `useMaterial3SliderShapes` ? I wonder if adding the year these were introduced in the specification is a good idea for the naming (or not)

Reply:
M3 shapes were already introduced, this is a new set.